subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/AbstractGradleExecuter.java

```diff
@@ -1439,6 +1439,19 @@ private void validate(String output, String displayName) {
             }
         }
 
+        /**
+         * Changes to a configuration's allowed usage contain the string "deprecated" and thus will trigger
+         * false positive identification as Deprecation warnings by the logic in {@link #validate(String, String)};
+         * this method is used to filter out those false positives.
+         *
+         * @param line the output line to check
+         * @return {@code true} if the line is a configuration allowed usage changing info log message; {@code false} otherwise
+         */
+        private boolean isConfigurationAllowedUsageChangingInfoLogMessage(String line) {
```

Comment:
I am skeptical of adding this false positive. If we are actually emitting a deprecation warning we should not blanket ignore it but instead fix either our tests to acknowledge it or avoid emitting the warning altogether. 

What caused you to need to add this? Did we switch the flag to warn on usage changes to true anywhere? I thought this PR was not going to do that.

Reply:
It's not a deprecation warning, it's a warning that happens to include the string "deprecation".  The regex that checks these in the tests is pretty simple.  There's nothing special about deprecation warnings vs. other warnings other than that string.